journal/modernized/14-chapter-12.adoc

```diff
@@ -409,7 +430,8 @@ of which I have, in his love, before treated; and in verses 15 and 16, he says,
 beware of false prophets, etc.
 You shall know them by their fruits, etc.
 
-TRUE MINISTERS
+[.blurb]
```

Comment:
I wasn't sure about this being a blurb, or about the capitalization.

Reply:
I think it looks good the way you did it. 👍🏻